### PR TITLE
cn: added node/cn/api_sc_backend_test.go

### PR DESCRIPTION
**Pull request by ehnuje**
## Proposed changes

- added `node/cn/api_sc_backend_test.go` to increase the coverage of `node/cn/api_sc_backend.go`

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: node/cn/api_sc_backend_test.go [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+}
+
+func TestServiceChainAPIBackend_SubscribeRemovedLogsEvent(t *testing.T) {
+	api, mockCtrl, mockBlockChain, _ := prepareServiceChainAPIBackendTest(t)
+	defer mockCtrl.Finish()
+
+	ch := make(chan<- blockchain.RemovedLogsEvent, 10)
+
+	mockBlockChain.EXPECT().SubscribeRemovedLogsEvent(ch).Times(1)
+	api.SubscribeRemovedLogsEvent(ch)
```
Comment (jeonghonah) [inline node/cn/api_sc_backend_test.go]: Is the purpose of this test to verify that no panic or hang occurs?

Comment (ehnuje) [reply node/cn/api_sc_backend_test.go]: It seems that I missed checking the return variable of functions. Thanks! I will update the tests.

Comment (ehnuje) [reply node/cn/api_sc_backend_test.go]: @jeonghonah After take a deeper look into the test, the purpose of these tests are to check if only the desired calls to the mockBlockChain happens or not. So it looks fine as it is.

**Comment by ehnuje** [comment]
@nogang @KimKyungup Dear reviewers, could you please review this change?

File: node/cn/api_sc_backend_test.go [hunk 2]
```diff
@@ -0,0 +1,402 @@
...
+
+func TestServiceChainAPIBackEnd_SetHead(t *testing.T) {
+	api, mockCtrl, mockBlockChain, _ := prepareServiceChainAPIBackendTest(t)
+	defer mockCtrl.Finish()
+
+	headNumber := uint64(12345)
+	mockBlockChain.EXPECT().SetHead(headNumber).Times(1)
+	api.SetHead(headNumber)
+}
+
```
Comment (KimKyungup) [inline node/cn/api_sc_backend_test.go]: Thanks. @ehnuje. :)
Will you add unit test of `HeaderByNumber`, `BlockByNumber`, and  `StateAndHeaderByNumber` in next PR?

Comment (ehnuje) [reply node/cn/api_sc_backend_test.go]: @KimKyungup Of course. Tackling easier ones first =)

**Pull request merged**